### PR TITLE
OS-13: setup tailwind css

### PR DESCRIPTION
**Pull request by rnkimani**
#### What does this PR do?
-  Sets up tailwind css as a function utility css framework
 #### How should this be manually tested?
- Run `yarn add ` then `yarn serve` the home page will be using this utilities
 #### Any background context you want to provide?
- I explored [tachyons](https://tachyons.io/)  vs [tailwind](https://tailwindcss.com) and chose tailwind for a couple of reasons
   - has verbose naming and easy to remember; eg `flex my-0` for display: flex and margin(y-axis): 0`
   - It has an `@apply` directive that enables "copying" of predefined css to your own custom css. This prevents lengthy dom elements like 
```html
<button
      class="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded"
    ></button>
```
   - Well structured documentation. See [docs](https://tailwindcss.com/components)

 #### What are the relevant Jira story IDs?
- [OS-13](https://gopuff.atlassian.net/jira/software/projects/OS/boards/33?selectedIssue=OS-13)

### CODE REVIEW

File: src/components/Example.vue [hunk 1]
```diff
@@ -15,4 +18,13 @@ export default {
 
 <style scoped lang="scss">
 @import "../styles/components/example";
+.btn {
+  @apply font-bold py-2 px-4 rounded mt-2;
```
Comment (rnkimani) [inline src/components/Example.vue]: This makes the style more readable and structured.

**Comment by KevinJMC** [comment]
I think you mean ```yarn``` or ```yarn add```, start is an npm commands

**Comment by rnkimani** [comment]
Yes, I updated the description

**Comment by mcoppola** [comment]
@rnkimani i'm ok with tailwind, thanks for the context

i think the core values we want to get are:
- rem based values
- box-sizing scales, instead of ad-hoc values (p-2, vs padding: 2rem)
- responsive shorthand

looks like we get these with both, the biggest difference i see is the verbosity in the short names.  i do agree that i like the @apply directive.  I would lean on using @apply in the .scss files vs creating really long class names in the markup.

lets move forward with this! thanks agian

**Comment by rnkimani** [comment]
Thanks!

**Pull request merged**